src/server/main_service.cc

```diff
@@ -2185,6 +2197,9 @@ void Service::Pubsub(CmdArgList args, ConnectionContext* cntx) {
         "\tReturn the currently active channels matching a <pattern> (default: '*').",
         "NUMPAT",
         "\tReturn number of subscriptions to patterns.",
+        "NUMSUB [<channel> ...]",
+        "\tReturn the number of subscribers for the specified channels, excluding",
+        "\tpattern subscriptions(default: no channels).",
```

Comment:
```suggestion
        "\tpattern subscriptions.",
```

(it's not really a matter of default-ness, it just works for the provided channels)

Reply:
fixed: https://github.com/dragonflydb/dragonfly/pull/2282/commits/aefd085665a47a0c0d72ae14961a5a4bc8555170